### PR TITLE
My new feature branch

### PR DESCRIPTION
**Pull request by sporter399**

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -64,11 +78,10 @@ export default {
   name: 'app',
   data() {
     return {
+      userLoggedIn: false,
+      userRegistrationActive: false,
       isModalVisible: false,
       currentEventId: '',
-      moment: moment,
```
Comment (pkurjanowicz) [inline client/src/App.vue:69]: We need moment: moment,

File: client/src/App.vue [hunk 2]
```diff
@@ -64,11 +78,10 @@ export default {
...
-      inputUserName: '',
```
Comment (pkurjanowicz) [inline client/src/App.vue:70]: We need inputUserName: '',

File: client/src/App.vue [hunk 3]
```diff
@@ -64,11 +78,10 @@ export default {
...
-      currentUser: '',
```
Comment (pkurjanowicz) [inline client/src/App.vue:71]: currentUser: '',

File: client/src/App.vue [hunk 4]
```diff
@@ -77,19 +90,64 @@ export default {
       eventResponseID: [],
       zippedEvent: [],
       sharedUsers: [], 
-      isModalVisible: false,
```
Comment (pkurjanowicz) [inline client/src/App.vue:80]: we need this: isModalVisible: false,

File: client/src/App.vue [hunk 5]
```diff
@@ -49,12 +52,23 @@
...
+  
 </template>
 
 <script>
 import axios from 'axios'
-import facebookLoginbutton from './components/facebookLogin.vue'
+import login from './components/login.vue'
+import register from './components/register.vue'
 import calendarView from './components/calendarView.vue'
+import VueTags from "vue-tags";
```
Comment (krskelton) [inline client/src/App.vue]: Remove import VueTags line

**Comment by sporter399** [comment]
Recommended changes made and pushed to my-new-feature-branch

SP

**Review by pkurjanowicz — APPROVED** [review]

**Pull request merged**